### DISCUSSION TITLE
[Feature Request] expose updateInkBar on TabMenu

### DISCUSSION THREAD

**Discussion by jumika** [primefaces/community#1885] [PrimeVue]
Most of the times the ink bar on TabMenu appears on the wrong place at initial render. I use the undocumented updateInkBar method to fix this. It would be nice to have this documented and exposed in the types.

**Comment by tugcekucukoglu** [comment]
How can we replicate the issue?

**Reply by jumika** [reply]
I'll try to reproduce this in a codepen this week.

**Reply by jumika** [reply]
There it is: https://stackblitz.com/edit/primevue-4-vite-issue-template-k72df2
You can trigger it by putting the Tabs inside a Dialog and make the default tab selection to anything but the default. When you open the Dialog, the ink bar will be misaligned.

**Reply by vajkone** [reply]
any update on this?
